inventory/byo/hosts.example

```diff
@@ -139,6 +139,13 @@ openshift_release=v3.7
 # Skip upgrading Docker during an OpenShift upgrade, leaves the current Docker version alone.
 # docker_upgrade=False
 
+# NOTE: This is tech preview and likely shouldn't be used
```

Comment:
it would be good if this captured the idea that an alternate container runtime is the primary runtime.

the current name implies a function that may make people think that sounds good, when it should only be used when docker is not the container runtime, but just used for builds.

Reply:
Good idea. Will rename it.